tokio/src/sync/watch.rs

```diff
@@ -854,6 +854,26 @@ impl<T> Drop for Receiver<T> {
 }
 
 impl<T> Sender<T> {
+    /// Creates the sending-half of the [`watch`] channel.
+    ///
+    /// See documentation of [`watch::channel`] for errors when calling this function.
+    /// Beware that attempting to send a value when there are no receivers will
+    /// return an error.
+    ///
+    /// [`watch`]: crate::sync::watch
+    /// [`watch::channel`]: crate::sync::watch
+    ///
+    /// # Examples
+    /// ```
+    /// let sender = tokio::sync::watch::Sender::new(0u8);
+    /// assert!(sender.send(3).is_err())
+    /// let _rec = sender.subscribe()
+    /// assert!(sender.send(4).is_ok())
```

Comment:
You will need to fix this test:
```

failures:

error: doctest failed, to rerun pass `-p tokio --doc`
---- tokio/src/sync/watch.rs - sync::watch::Sender<T>::new (line 882) stdout ----
error: expected `;`, found keyword `let`
 --> tokio/src/sync/watch.rs:884:33
  |
5 | assert!(sender.send(3).is_err())
  |                                 ^ help: add `;` here
6 | let _rec = sender.subscribe()
  | --- unexpected token

error: expected `;`, found `assert`
 --> tokio/src/sync/watch.rs:885:30
  |
6 | let _rec = sender.subscribe()
  |                              ^ help: add `;` here
7 | assert!(sender.send(4).is_ok())
  | ------ unexpected token

error: aborting due to 2 previous errors

Couldn't compile the test.

failures:
    tokio/src/sync/watch.rs - sync::watch::Sender<T>::new (line 882)
```
Also, please add \`\`\` at the end of the example as well.

Reply:
Should be fixed now